netbox_secrets/views.py

```diff
@@ -393,11 +394,20 @@ def post(self, request):
         if form.is_valid():
             master_key = self.userkey.get_master_key(form.cleaned_data['secret_key'])
             user_keys = form.cleaned_data['user_keys']
+
             if master_key:
                 for user_key in user_keys:
-                    user_key.activate(master_key)
-                    messages.success(request, f"Successfully activated {len(user_keys)} user keys.")
-                    return redirect("plugins:netbox_secrets:userkey_list")
+                    target_user = user_key.user
```

Comment:
This seems like half a solution. While it might work on the UI form, the API will have the same issue.